src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -48,6 +49,32 @@
     (normalize-input current v)
     current))
 
+(defn- routes
+  [{:keys [amount from-network to-network]}]
+  [rn/view {:style style/routes-container}
+   [rn/view {:style style/routes-header-container}
+    [quo/section-label
+     {:section         (i18n/label :t/from-label)
+      :container-style style/section-label}]
+    [quo/section-label
+     {:section         (i18n/label :t/to-label)
+      :container-style (merge style/section-label {:margin-left 64})}]]
```

Comment:
imo or situations like this it's better to make `section-label` a function and take "margin-left" as a param